doc/plugins.md

```diff
@@ -2,33 +2,33 @@
 
 # Plugins
 
-**remark** is a markdown processor powered by plugins part of the [unified][]
-[collective][].
+**remark** is a tool that transforms markdown with plugins.
+See [the monorepo readme][remark] for info on what the remark ecosystem is.
+This page lists existing plugins.
 
 ## Contents
 
 *   [List of plugins](#list-of-plugins)
 *   [List of presets](#list-of-presets)
 *   [List of utilities](#list-of-utilities)
-*   [Using plugins](#using-plugins)
-*   [Creating plugins](#creating-plugins)
+*   [Use plugins](#use-plugins)
+*   [Create plugins](#create-plugins)
 
 ## List of plugins
 
-See [awesome remark][awesome] for the most awesome projects in the ecosystem.
+See [`awesome-remark`][awesome-remark] for the most awesome projects in the
+ecosystem.
 More plugins can be found on GitHub tagged with the
 [`remark-plugin` topic][topic].
 
-Have a good idea for a new plugin?
-See [Creating plugins][create] below.
+> 👉 **Note**: some plugins don’t work with recent versions of remark due to
```

Comment:
A link here is already referenced in the migration guide: https://github.com/remarkjs/remark/releases/tag/13.0.0. That’s probably a better place for folks to find this point instead of buried somewhere in a big monorepo readme?

good point that folks are still upgrading though!